packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -347,26 +347,17 @@ describe( 'Navigation editor', () => {
 		);
 		await startEmptyButton.click();
 
-		// NOTE - the following code is commented out.
-		// In CI the editor doesn't seem to support variations.
-		// The following code can be re-introduced once that's resolved.
-		// Add an inner link block.
-		// const appender = await page.waitForSelector(
-		// 	'button[aria-label="Add block"]'
-		// );
-		// await appender.click();
+		const appender = await page.waitForSelector(
```

Comment:
@gwwar Thanks for switching this back. I'm curious - did the underlying issue (with variations not being available) get fixed here, or did it fix itself?

Reply:
Maybe from the server-side piece landing? https://github.com/WordPress/wordpress-develop/pull/1015#issuecomment-797576466